starknet_py/utils/typed_data/typed_data.py

```diff
@@ -1,7 +1,5 @@
-# pyright: reportGeneralTypeIssues=false
-
 from dataclasses import dataclass
```

Comment:
Why have you moved the file? This is a breaking change and import looks bad (from ...typed_data.typed_data).

Reply:
The convention is to create files with unit tests next to the implementation, but the test file in the previous directory created circular import error.

Also I would like to import typed_data into the `__init__.py` file of `typed_data` module, so it can be used easier (from ...typed_data import TypedData, instead of from ...typed_data.typed_data import TypedData), but @cptartur noticed, we've had previous problems with imports in `__init__.py` files and should avoid it.

What do you think?

Reply:
So after an investigation, it seems that the initial circular import error was created because of our file names `types.py`, which is the same as some **python internal file**. Changing the name of our `types.py` resolves the issue.

Reply:
I don't see how typed_data.py could work without circular import, but typed_data_test.py couldn't and I don't see how types.py is related to this.

I've just movded the file typed_data.py and typed_data_test.py to utils root and they work properly, all tests pass.